lib/sandox-utils.js

```diff
@@ -6,20 +6,20 @@ let SANDBOX = Symbol();
 // use this one if using a mix of mocks/stub/spies
```

Comment:
should we take the time to rename this file since you're in here?

Reply:
Rename to what? Not sure what you mean.